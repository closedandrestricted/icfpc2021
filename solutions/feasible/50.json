{"vertices":[[29,42],[51,13],[27,3],[50,23],[10,63],[29,26],[30,43],[24,20],[45,8]]}