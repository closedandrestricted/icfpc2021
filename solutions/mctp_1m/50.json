{"vertices":[[10,65],[42,15],[16,18],[30,45],[15,93],[29,37],[10,65],[33,71],[48,52]]}