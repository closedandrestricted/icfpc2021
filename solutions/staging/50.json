{"vertices":[[15,82],[24,52],[46,38],[20,54],[20,54],[48,61],[35,30],[46,51],[24,61]]}