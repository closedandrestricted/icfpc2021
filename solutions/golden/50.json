{"vertices":[[0,22],[13,88],[11,62],[25,35],[25,35],[32,71],[36,9],[50,28],[47,52]]}